webapp/babel.cfg

```diff
@@ -1,2 +1,3 @@
 [python: app/**.py]
-[jinja2: app/templates/**.html]
\ No newline at end of file
+[jinja2: app/templates/**.html]
+extensions=jinja2.ext.autoescape,jinja2.ext.with_
```

Comment:
extension still neded - the new version claims that this is now in the core but this never worked so far